instances/atlas/brainAtlasVersion/WHSSDatlas_v1.jsonld

```diff
@@ -254,7 +254,7 @@
         "ontologyIdentifier": null
     },
     "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
-    "howToCite": "Please refer to the atlas by its RRID:SCR_017124, and cite the following publications: <a href='https://doi.org/10.1016/j.neuroimage.2014.04.001'>Papp et al (2014)</a>",
+    "howToCite": "Please refer to the atlas by its <a href='https://scicrunch.org/resolver/SCR_017124'>RRID:SCR_017124</a>, and cite the following publications: <a href='https://doi.org/10.1016/j.neuroimage.2014.04.001'>Papp et al (2014)</a>",
```

Comment:
please use as I posted in my comments markdown syntax.

"Please refer to the atlas by its [RID:SCR_017124](https://scicrunch.org/resolver/SCR_017124), and cite the following publications: [Papp et al (2014)(https://doi.org/10.1016/j.neuroimage.2014.04.001)"

Reply:
> 
I do not know how to write in markdown syntax, can you post exactly how you want me to write it in the document. I tied with <a href = ''> but i cannot use "" as it stops the quote. I have tried fo copy paste what you wrote, but the hyperlink is not conserved. Also do you mean to close the [.

Reply:
`"Please refer to the atlas by its [RID:SCR_017124](https://scicrunch.org/resolver/SCR_017124), and cite the following publications: [Papp et al (2014)](https://doi.org/10.1016/j.neuroimage.2014.04.001)"`

Reply:
I already did... here again

Reply:
```json
"howToCite": "Please refer to the atlas by its [RRID:SCR_017124](https://scicrunch.org/resolver/SCR_017124), and cite the following publications: [Papp et al (2014)](https://doi.org/10.1016/j.neuroimage.2014.04.001)"
```

Reply:
> I already did... here again

Sorry, the full format showed up now, i am not quite sure why it was displayed different earlier. Anyway, i pushed the change at v1 brain atlas version, is it correct now?